index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Quick startup</title>
+</head>
+
+<body>
+    <!--This wraps on my page together-->
+    <div class="container">
+        <header class="header-page">
+            <!--header contents goes here-->
+
+            <nav>
+                <!--This is my Logo-->
+                <h3>Solorics</h3>
+
+                <!--Toggle button goes here-->
+                <img src="images/Union.png" alt="Toggle_Image_Button" class="toggle">
+            </nav>
+        </header>
+        <main class="main-page">
+            <!--main contents goes here-->
+            <div>
+                <h3>Opeyemi</h3>
```

Comment:
- [ ] Please change the main title to `h1` tag. 
If you want that google (and other search engines) knows about a specific topic, in this case, you, use an `h1` tag. So basically we do this for SEO reasons . cheers 👍